rasa/nlu/extractors/extractor.py

```diff
@@ -254,6 +268,40 @@ def _update_confidence_values(
                 confidences[ENTITY_ATTRIBUTE_GROUP][idx],
             )
 
+    @staticmethod
+    def _check_is_single_entity(
+        text: Text,
+        token: Token,
+        last_token_end: int,
+        split_entities_config: Dict[Text, bool],
+        current_entity_tag: Text,
+    ):
+        # Tokens need to be no further than 3 positions apart
+        tokens_within_range = token.start - last_token_end <= 3
+
+        # The interleaving tokens *must* be a full stop, a comma, or a whitespace
```

Comment:
what about other symbols like `;`?

Reply:
Should we maybe move the allowed character set to a constant somewhere, so it can be extended more easily? I can certainly add a semi-colon.